### PR TITLE
Add test case to raise valu description

### PR DESCRIPTION
**Pull request by lizimu0709**
Add the check in the sv_harmonic_mean if the DataFrame is empty before trying to access its columns and the corresponding test case from https://github.com/uw-ssec/offshore-geodesy/issues/105#issue-1827136610.

### CODE REVIEW

**Comment by lsetiawan** [comment]
@lizimu0709 Please fix the merge conflict before proceeding. Seems like the tests are not running because there's a merge conflict. Thanks.

File: src/gnatss/harmonic_mean.py [hunk 1]
```diff
@@ -71,6 +73,8 @@ def sv_harmonic_mean(svdf: pd.DataFrame, start_depth: float, end_depth: float) -
     abs_sv = abs(svdf)
     # Get the index for the start of depth closest to specified start depth
     if len(abs_sv) == 0:
-        raise ValueError("Dataframe is empty! Please check your data inputs.")
+        raise ValueError(
+            "DataFrame with empty 'abs_sv' column! Please check your data inputs."
```
Comment (lsetiawan) [inline src/gnatss/harmonic_mean.py:77]: `abs_sv` is the same exact dataframe but values are the absolute value, not an empty 'abs_sv' column. If you see line 73 above, it wraps `svdf` with `abs` function. You've caught this with your new change in `svdf.empty` so probably don't need this check anymore of `if len(abs_sv)`... does that makes sense?

Comment (lizimu0709) [reply src/gnatss/harmonic_mean.py:77]: Thanks for noting this redundancy issue, I am sorry for didn't notice this. I will also delete the redundancy test case in the test file too.

**Review by lizimu0709 — COMMENTED** [review]
Removed the redundant check part in harmonic_mean.py and test_harmonic_mean.py

**Review by lsetiawan — APPROVED** [review]

**Pull request merged**